docs/pages/example/map-tiles.html

```diff
@@ -8,8 +8,9 @@
         "sources": {
             "raster-tiles": {
                 "type": "raster",
-                "url": "mapbox://" + tileset,
-                "tileSize": 256
+                "tiles": ["http://tile.stamen.com/watercolor/{z}/{x}/{y}.jpg"],
+                "tileSize": 256,
+                "attribution": '<a id="home-link" target="_top" href="../" class="">Map tiles</a> by <a target="_top" href="http://stamen.com">Stamen Design</a>, under <a target="_top" href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a target="_top" href="http://openstreetmap.org">OpenStreetMap</a>, under <a target="_top" href="http://creativecommons.org/licenses/by-sa/3.0" class="">CC BY SA</a>' 
```

Comment:
how about this?

`Map tiles by <a target="_top" rel="noopener" href="http://stamen.com">Stamen Design</a>, under <a target="_top" rel="noopener" href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a target="_top" rel="noopener" href="http://openstreetmap.org">OpenStreetMap</a>, under <a target="_top" rel="noopener" href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>`